understanding/22/target-size-minimum.html

```diff
@@ -44,7 +44,7 @@ <h3>Size requirement</h3>
 
             <figure id="target-size-basic">
                 <img src="img/target-size-basic.svg" width="180" height="75" alt="Three square boxes next to each other, each with a height and width of 24px">
-                <figcaption>Where targets are a 24 by 24px square (and larger is better), they meet the size requirement of the criterion and pass (image shown to scale - <a href="img/target-size-basic.svg">see the scalable original version</a>)</figcaption>
+                <figcaption>Where targets are at least a 24px by 24px square, they meet the size requirement of the criterion and pass (image shown to scale - <a href="img/target-size-basic.svg">see the scalable original version</a>)</figcaption>
```

Comment:
```suggestion
                <figcaption>Where targets are at least a 24 by 24 CSS pixel square, they meet the size requirement of the criterion and pass (image shown to scale - <a href="img/target-size-basic.svg">see the scalable original version</a>)</figcaption>
```

@mbgower i checked through the whole understanding, and we already consistently use "24 by 24 CSS pixel", so changing it to that